src/config/URLConfig.ts

```diff
@@ -13,6 +13,7 @@ class URLConfig extends Config implements IURLConfig {
   useRootPath?: boolean;
   secure?: boolean;
   forceVersion?: boolean;
+  urlSearchParams?: Record<string, string | number | boolean>;
```

Comment:
Maybe just call it `queryParams` ?

Reply:
actually it was `queryParams` but we wanted to be aligned wit web api
https://github.com/cloudinary/js-url-gen/pull/555#discussion_r1151699317

Reply:
@lbendavid-cloudinary, @pawelphilipczyk-cloudinary 
Is Web API very popular? I hear this name for the first time. And we will have to align this implementation with other SDKs(at some point). In all other languages it is called query parameters/query string.

Reply:
The class is called URLConfig, having `url` prefix in `urlSearchParams` looks redundant, no matter what name we choose, it should be without `url` prefix.

Reply:
@const-cloudinary It's not about popularity, `URLSearchParams` is the standard name for the [interface in Web API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). People call it query params and I did too until learning this. 

If we want to align with other SDKs then I take that as a fair point and back down with my initial comments.

Reply:
The first line you read in the description is:
`defines utility methods to work with the query string of a URL.`
And in RFC the component is called "Query":
https://www.rfc-editor.org/rfc/rfc3986#section-3.4

But if you insist, I'm OK with that.

Reply:
I think having a consistent naming across our SDK's is important, also from docs point of view. So i think `queryParams` would be the better name